src/graphql/mirror.js

```diff
@@ -318,6 +318,18 @@ export class Mirror {
           CREATE INDEX idx_connection_entries__connection_id
           ON connection_entries (connection_id)
         `,
+        dedent`\
+          CREATE TABLE network_log (
```

Comment:
Table schema looks good! You are creating a new table, so you must bump
the Mirror schema version (currently `MIRROR_v7`).

Reply:
updated